frontend/src/lib/ui-text.js

```diff
@@ -377,9 +377,9 @@ const uiText = {
     mobileLabelAdm: "Administrations",
     mobileLabelForms: "Forms",
     mobileNameRequired: "Please input name",
-    mobileAdmRequired: "Please select one or more villages",
+    mobileAdmRequired: "Please select one or more Administrative level name",
     mobileFormsRequired: "Please select one or more forms",
-    mobileSelectAdm: "Select village...",
+    mobileSelectAdm: "Select Administrative level name...",
```

Comment:
What I mean by Administration Level name is the name of administration, depends on the previous selection.

Reply:
if it's hard to achieve, just make the warning select one or more administration